### PR TITLE
Auth Setup

### PR DESCRIPTION
**Pull request by GuptaEesh**
- Login
  - With error handler
  - Guest Login
  - a loader to get you started

- Sign-Up
  - With error handler
  - With Password checks
  - a loader to get you started
  
 
login
signup

### CODE REVIEW

**Review by shivampandey0 — APPROVED** [review]
- Good use of barrel imports
- Good use of dynamic selection of Input fields

Good to merge

File: src/pages/signup/signup.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    error: false,
+    message: "",
+    loader: false,
+  };
+  const navigate = useNavigate();
+  const { login, isAuthenticated } = useAuth();
+  const [formFields, setFormFields] = useState(initial);
+  const { name, email, password, confirmPass, error, message, loader } =
+    formFields;
+  console.log(formFields);
```
Comment (AnuragSahu11) [inline src/pages/signup/signup.js:19]: console.log can be removed

**Review by AnuragSahu11 — APPROVED** [review]
Good use of location to display sidebar, you can merge

File: src/pages/signup/signup.js (hunk 1 above)
Comment (GuptaEesh) [reply src/pages/signup/signup.js:19]: Yes, I have removed them, were used for testing. Thanks for pointing that out

**Pull request merged**